buildSrc/src/main/groovy/nva.identity.service.jacoco-merge.gradle

```diff
@@ -0,0 +1,66 @@
+//For generating one unified coverage report
```

Comment:
the jacoco-merge should be applied only on the top project. Therefore I put the logic in a single plugin and used the plugin in the main build.gradle file